doc/i2p.md

```diff
@@ -0,0 +1,64 @@
+# I2P SUPPORT IN BITCOIN
+
+It is possible to use Bitcoin Core to accept and make connections from/to other
```

Comment:
```suggestion
It is possible to use Bitcoin Core to accept and make connections to/from other
```

Reply:
Since the wording is "accept from" and "connect to" and since "accept" comes before "connect", I put "from" before "to". Reworded in a different way now.